docs/source/models/causes/neonatal/index.rst

```diff
@@ -29,45 +29,231 @@ the strategy for capturing the burden of the neonatal subcauses to calibrate to
 
 GBD 2021 Modeling Strategy
 --------------------------
 
+I guess this comes from the mortality team.
+
 Cause Hierarchy
 +++++++++++++++
 
-Subcause definitions
-""""""""""""""""""""""""""""
+- All causes (c_294) [level 0]
+
+  - Communicable, maternal, neonatal, and nutritional diseases (c_295)
+
+    - Maternal and neonatal disorders (c_962)
+
+      - **Neonatal disorders (c_380)** [level 3]
+
+
+Modeled Subcauses
+-----------------
+
+.. toctree::
+    :maxdepth: 1
+
+    neonatal_sepsis
+    neonatal_encephalopathy
+    preterm_birth
+
 
 Restrictions
 ++++++++++++
 
+The following table describes the restrictions from GBD 2021 and our intended use in our MNCNH model.
+
+.. list-table:: GBD 2017 Cause Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - True
+     - **Note:** GBD estimates both YLLs and YLDs for neonatal disorders, but
+       **we are ignoring YLDs for this model**.
+   * - YLD only
+     - True
+     - See above.
+   * - YLL age group start
+     - Early Neonatal
+     - [0, 7 days), age_group_id=2
+   * - YLL age group end
+     - Late Neonatal
+     - [7 days, 28 days), age_group_id=3
+   * - YLD age group start
+     - Not applicable
+     - 
+   * - YLD age group end
+     - Not applicable
+     - 
+
+
+
 Vivarium Modeling Strategy
 --------------------------
+This model is designed to estimate deaths and YLLs due to neonatal disorders that could be
+averted by a suite of intervention that affect sepsis, respiratory stress disorder, and perhaps encephalopathy, as well as :ref:`Low Birth Weight and Short
+Gestation (LBWSG) <2019_risk_effect_lbwsg>`. The model groups together all of the
+neonatal sub-causes, and is fatal-only (no disability). The rationale for this
+design is as follows:
+
+1. The LBWSG risk factor in GBD acts on overall neonatal mortality, so we need to model all-cause neonatal mortality to calibrate to all-cause mortality, preterm paf-of-one, and LBWSG risk.
+
+2. The nonfatal burden of neonatal causes is around 10% of the total burden, so we will have a much simpler model without missing too much by excluding YLDs.
 
 Scope
 +++++
 
+1. Capture neonatal deaths, and the relationship with LBWSG.
+2. Capture the deaths averted by interventions that reduce the cause-specific mortality rates of preterm with respiratory distress and sepsis (and perhaps encephalopathy).
+3. Do not capture nonfatal burden.
+
+
 Assumptions and Limitations
 +++++++++++++++++++++++++++
 
+The excluded nonfatal portion of the burden is small (around 10% of the total burden).
+
+The evidence base for identifying the level 4 subcauses is not as solid as I would like.
+
+Preterm must be split into with and without respiratory distress syndrome (RDS) outside of GBD, since GBD does not distinguish preterms deaths with and without RDS.
```

Comment:
Do we have a data strategy for RDS beyond using the preterm birth cause from GBD?

Reply:
And will we be documenting this on the preterm birth subcause model page?

Reply:
Good questions. In regards to the first one, in the [nanosim](https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/blob/main/lbwsg_nanosim/2024_09_17_ai_ultrasound_pathway_nanosim.ipynb), I [used Table 4](https://github.com/ihmeuw/vivarium_research_mncnh_portfolio/blob/main/lbwsg_nanosim/preterm_cod.py) from [this paper](https://www.sciencedirect.com/science/article/pii/S2214109X19302207) (emailed to us by Chris Troeger) to estimate that about 85% of preterm birth deaths are due to RDS. We could use the same table to get percentages stratified by gestational age bins as well. However, this paper is from a single study in Ethiopia -- I don't know where to find more wide-ranging data.